awscli/customizations/codeartifact/login.py

```diff
@@ -78,21 +78,27 @@ def _run_commands(self, tool, commands, dry_run=False):
             return
 
         for command in commands:
-            try:
-                self.subprocess_utils.check_call(
-                    command,
-                    stdout=self.subprocess_utils.PIPE,
-                    stderr=self.subprocess_utils.PIPE,
-                )
-            except OSError as ex:
-                if ex.errno == errno.ENOENT:
-                    raise ValueError(
-                        self._TOOL_NOT_FOUND_MESSAGE % tool
-                    )
-                raise ex
+            self._run_command(tool, command)
 
         self._write_success_message(tool)
 
+    def _run_command(self, tool, command, *, ignore_errors=False):
```

Comment:
I don't think the `*` is going to be doing anything here. If we needed to change this function in the future, any new parameters would need to be added to the end of the function for compatibility (even though this is technically a private method).

Is there something that I'm missing that this is needed for?

Reply:
`*` makes it keyword parameter rather than positional. So any new positional param would need to add before * and any new required keyword param after it based on requirement. Are you suggesting it to be not keyword param?